### PR TITLE
Adds workaround for electron dev tools shortcut

### LINKED ISSUE #87: Enable keyboard shortcut to open DevTools
If I start the Electron app and close de DevTools, there is no way to open it again. Normally (on Mac) the keyboard shortcut should be CMD+OPT+I but it currently doesn't do anything. Probably this doesn't work because Theia has its own contribution system that overrides this.

### PR DESCRIPTION
**Pull request by epatpol**
Fixes #87

Signed-off-by: Patrick-Jeffrey Pollo Guilbert <patrick.pollo.guilbert@ericsson.com>

### CODE REVIEW

**Comment by epatpol** [comment]
Is that what you meant by hardcoding it @hexa00 ? Can somebody on mac test this also? Thanks!

**Comment by hexa00** [comment]
Yes that is what I meant... I'll test it on windows too tomorrow

**Comment by kittaakos** [comment]
It does not work on OS X. I did the following:
```
git clone https://github.com/epatpol/theia-1.git \
&& cd theia-1 \
&& git checkout origin/87 -b 87 \
&& yarn --unsafe-perm \
&& yarn run rebuild:electron \
&& cd examples/electron \
&& yarn run start
```
It does not do anything on `Cmd+Shift+I`. I have tried it with `Ctrl+Shift+I` too, it did not work either.

**Comment by epatpol** [comment]
Thanks for testing it @kittaakos, although I'm not sure what's wrong here :/

**Comment by epatpol** [comment]
Weird, would replacing CommandOrControl+Shift+I by Command+Shift+I make it work on mac os (just sanity checking here)

**Comment by hexa00** [comment]
I tested on windows, on chome and firefox it works.

I have not been able to try with electron yet.

**Comment by kittaakos** [comment]
This change does not affect the browser at all. At least it should not. So the question is still the electron version.

**Comment by hexa00** [comment]
Oops indeed my bad.

I just tested the electron version now on windows and it works

**Comment by epatpol** [comment]
@kittaakos @hexa00 can you guys test again (both on Windows/Mac OS) if you have time as I completely changed how it's done (currently using theia framework instead of the globalShortcuts for electron). Thanks!

**Comment by kittaakos** [comment]
Works cool on OS X with your latest changes. One small remark; when the `dev-tools` is not visible, one can open it with Cmd+Shift+I on OS X. Works cool. Once I have opened it, the `dev-tools` grabs the focus, and I have to set the focus into `theia` if I would like to close it with the key bindings again.

**Comment by kittaakos** [comment]
I have also restarted the Windows CI build, there was a problem during the `yarn` installation phase.

File: packages/core/src/electron-browser/menu/electron-commands.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ */
+
+import { injectable } from "inversify"
+import { CommandContribution, CommandRegistry } from "../../common"
+import * as electron from "electron"
+
+@injectable()
+export class ElectronCommandHandlers implements CommandContribution {
+
+    constructor(
```
Comment (kittaakos) [inline packages/core/src/electron-browser/menu/electron-commands.ts]: Do we really need this `ctor` function here?

Comment (epatpol) [inline packages/core/src/electron-browser/menu/electron-commands.ts]: You're right, let me delete it

**Comment by epatpol** [comment]
> Works cool on OS X with your latest changes. One small remark; when the dev-tools is not visible, one can open it with Cmd+Shift+I on OS X. Works cool. Once I have opened it, the dev-tools grabs the focus, and I have to set the focus into theia if I would like to close it with the key bindings again.

@kittaakos I'm not exactly sure how to fix this with the theia keybindings api at the moment, I can take a look, otherwise I was thinking about adding a "Toggle Developer Tools" menu item like in VSCode. I think by default they don't have any shortcut for it in VSCode afaik.

File: packages/core/src/electron-browser/menu/electron-commands.ts (hunk 1 above)
Comment (kittaakos) [inline packages/core/src/electron-browser/menu/electron-commands.ts]: Thanks! Other than this, looks great and works correctly, so please merge once you are done.

**Comment by kittaakos** [comment]
> I'm not exactly sure how to fix this with the theia keybindings api at the moment

I would just create a follow-up task.

**Pull request merged**